src/test/java/teammates/test/cases/common/SanitizerTest.java

```diff
@@ -44,27 +47,62 @@ public void testSanitizeTextField() {
     
     @Test
     public void testSanitizeForJs() {
-        
+
+        String unsanitized = "\\ \" ' #"; // i.e., [\ " ' #]
+        String expected = "\\\\ \\&quot; \\&#39; \\#"; // i.e., [\\ \&quot; \&#39; \#]
+        String sanitized = Sanitizer.sanitizeForJs(unsanitized);
+
+        assertEquals(null, Sanitizer.sanitizeForJs(null));
+        assertEquals(expected, sanitized);
     }
     
     @Test
     public void testSanitizeForHtml() {
-        
+
+        String unsanitized = "< > \" / ' &"
+                           + "<script>alert('injected');</script>";
+
+        String expected = "&lt; &gt; &quot; &#x2f; &#39; &amp;"
+                        + "&lt;script&gt;alert(&#39;injected&#39;);&lt;&#x2f;script&gt;";
+
+        String sanitized = Sanitizer.sanitizeForHtml(unsanitized);
+        String sanitizedTwice = Sanitizer.sanitizeForHtml(sanitized);
+
+        assertEquals(null, Sanitizer.sanitizeForHtml(null));
```

Comment:
This comment applies to all the different tests that you've written - can we break up the different tests within a parent test into "lesser test methods" so it is more modularized and it's immediately apparent what each "lesser test method" is trying to test for 

For instance, ```assertEquals(null, Sanitizer.sanitizeForHtml(null));``` would go under something like ```testSanitizeForHtmlReceivesNullReturnsNull```

and so on, single sanitization, double sanitization etc.